### PR TITLE
[ci] Add short benchmarks to the pipeline

### PR DESCRIPTION
**Pull request by alvicsam**
The PR adds short benchmarks and removes `check-runtime-benchmarks` job as suggested in  https://github.com/paritytech/ci_cd/issues/353

### CODE REVIEW

**Comment by shawntabrizi** [comment]
Is the intention to remove so many of the other CI pipelines here?

**Review by sergejparity — APPROVED** [review]

File: scripts/run_short_benches_for_runtime.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+echo "[+] Running all benchmarks for $runtime"
+
+./artifacts/polkadot benchmark \
+    --chain "${runtime}-dev" \
+    --list |\
+  tail -n+2 |\
+  cut -d',' -f1 |\
+  uniq | \
+  grep -v frame_system > "${runtime}_pallets"
```
Comment (coderobe) [inline scripts/run_short_benches_for_runtime.sh:19]:
```suggestion
  uniq > "${runtime}_pallets"
```
to be in sync with https://github.com/paritytech/polkadot/commit/5d9dc9bd1b95e9c64cfc547f45f0b30e7cc03097

**Review by coderobe — APPROVED** [review]

**Review by ggwpez — APPROVED** [review]
Wow very cool! Nearly suspicious that they all succeed :laughing:  

@brunopgalvao You can incorporate rococo here as well, once rococo benches work.

**Comment by ggwpez** [comment]
I double checked and something is not quite right, but we can probably already merge this and fix that later:

The `ump` benchmark still does not work, but the script is not catching it since it seems to not execute it:  
```sh
polkadot benchmark --execution wasm --wasm-execution compiled --chain polkadot-dev --pallet "runtime_parachains::ump" --extrinsic "*" --steps 1 --repeat 1
```
We can probably replace the script with just:  
```sh
polkadot benchmark --execution wasm --wasm-execution compiled --chain ${runtime}-dev --pallet "*" --extrinsic "*" --steps 1 --repeat 1
```

**Comment by ggwpez** [comment]
Yep now it detects the error, thanks for changing it. Maybe you can make it "allowed to fail" until we fix it?

**Comment by ggwpez** [comment]
bot rebase

**Comment by alvicsam** [comment]
@ggwpez it seems that rebase didn't help. Should I make short-benchmarks `allow_fail: true` ?

**Comment by ggwpez** [comment]
> it seems that rebase didn't help. Should I make short-benchmarks `allow_fail: true` ?

It helped, but now a different one fails :facepalm: (`pallet-bags-list`). @alvicsam   
Let's make it "allowed to fail" and the change it back once we either fixed them or added a `--exclude-pallet`.

**Pull request merged**